en/telemetry/microhard_serial.md

```diff
@@ -0,0 +1,14 @@
+# Microhard Serial Radio
+
+[Microhard Pico Serial](http://microhardcorp.com/P900.php) radios enable MAVLink communication between a radio on a vehicle and a GCS.  
+Microhard Pico Serial radios are 1 watt output radios that support point to point, point to multi-point, and mesh modes.
+A single ground station radio can be used to communicate with multiple vehicles using point to multi-point or mesh. Vehicles must have different IDs.
+
+![Microhard Radio](../../assets/hardware/telemetry/ark_microhard_serial.jpg)
```

Comment:
Just FMI, why prefix "ark" on this file?

Reply:
Its a picture of my microhard product. 
https://arkelectron.com/product/1w-900mhz-serial-telemetry-air-radio/